examples/starter/src/components/entry-point/entry-point.spec.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
could we use react-testing-library instead? I would leave out enzyme entirely.

Reply:
I did (`main-view.spec.js`), but I also wanted to have a simple example of using shallow rendering.